index.html

```diff
@@ -3,21 +3,25 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
     <title>Fetch</title>
   </head>
   <body>
     <section id="recipe-section">
       <h2 class="section-heading">Welcome to the Recipe Finder!</h2>
       <input
+        placeholder="Feeling hungry stranger? Just type here any ingredient name and I'll find you someething tasty!"
         aria-label="food-input"
```

Comment:
A good use of an aria-label here. You can also use 'title' which will display the message on mouseover.